session.go

```diff
@@ -99,6 +103,9 @@ type SessionHeartbeatHandler func(ctx context.Context, sess Session, latency tim
 // ServerCommandHandler is the callback type for [WithStopHandler]
 type ServerCommandHandler func(ctx context.Context, sess Session) error
 
+// StopTunnelHandler is the callback type for [WithStopTunnelHandler]
+type StopTunnelHandler func(ctx context.Context, sess Session, clientID string, err error) error
```

Comment:
Do we need a user-assignable handler for this? What do you get out of this that you wouldn't be able to discern from the `Tunnel.Accept` method returning an error?

If we _were_ to allow users to set this, it feels a little odd that we still close the tunnel outside of the user-provided handler. It differs from the other handlers that we offer (Stop, Restart, Update), which are no-ops if the user doesn't set them. Seems unexpected that this one doesn't actually let you change the behavior.

Reply:
Removed! User can't force a hot reconnect now, which was the main reason for it to exist.